sdk/zksend/src/links/builder.ts

```diff
@@ -230,26 +222,26 @@ export class ZkSendLinkBuilder {
 		// Ensure that gas amount ends in 987
 		const roundedGasAmount = gasWithBuffer - (gasWithBuffer % 1000n) - 13n;
 
-		const address = this.#keypair.toSuiAddress();
+		const address = this.keypair.toSuiAddress();
 		const objectsToTransfer = (await this.#objectsToTransfer(txb)).map((obj) => obj.ref);
 		const [gas] = txb.splitCoins(txb.gas, [roundedGasAmount]);
 		objectsToTransfer.push(gas);
 
-		txb.setSenderIfNotSet(this.#sender);
+		txb.setSenderIfNotSet(this.sender);
 		txb.transferObjects(objectsToTransfer, address);
 
 		return txb;
 	}
 
 	async #estimateClaimGasFee(): Promise<bigint> {
```

Comment:
Just to confirm this is only used in the old link style right?

Reply:
correct